src/cpp/session/modules/SessionFind.cpp

```diff
@@ -1256,6 +1276,11 @@ core::Error runGrepOperation(const GrepOptions& grepOptions, const ReplaceOption
    if (grepOptions.gitFlag())
    {
       cmd = shell_utils::ShellCommand("git");
+      cmd << "-c" << "grep.lineNumber=true";
```

Comment:
Might be worth adding a comment indicating that we set these configuration options to override any potential user-defined options?

Reply:
Good point, comment added!